CHANGELOG

```diff
@@ -9,6 +9,9 @@
   - Bugs fixes:
     - Evidence: Prevent loading old Evidence template content at the Issue level
     - Methodologies: validate presence of content
+    - Burp HTML: Upload file without issues
```

Comment:
I'd argue that this doesn't really solve the bug. What about just adding `nokogiri` to the `Upgraded gems` section?

Reply:
Yeah, i think its better. Moved it